src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -60,11 +60,10 @@ public async Task Expect100Continue_WaitsExpectedPeriodOfTimeBeforeSendingConten
                     var request = new HttpRequestMessage(HttpMethod.Post, uri) { Content = content, Version = UseVersion };
                     request.Headers.ExpectContinue = true;
 
-                    var sw = Stopwatch.StartNew();
+                    long start = Environment.TickCount64;
                     (await invoker.SendAsync(TestAsync, request, default)).Dispose();
-                    sw.Stop();
-
-                    Assert.InRange(sw.Elapsed, delay - TimeSpan.FromSeconds(.5), delay * 20); // arbitrary wiggle room
+                    long elapsed = content.Ticks - start;
+                    Assert.True(elapsed >= delay.TotalMilliseconds);
```

Comment:
You might want to have a small amount of leeway here in checking that elapsed >= delay, just in case of timing weirdness -- looks like the test previously had 0.5s of leeway?

If it's reliable as is, then maybe no leeway is necessary.